package.json

```diff
@@ -44,6 +44,18 @@
     "@sentry/node": "7.25.0",
     "@sentry/tracing": "7.25.0",
     "@sentry/types": "7.25.0",
+    "@opentelemetry/api": "1.3.0",
+    "@opentelemetry/auto-instrumentations-node": "0.35.0",
+    "@opentelemetry/core": "1.8.0",
+    "@opentelemetry/exporter-trace-otlp-proto": "0.34.0",
+    "@opentelemetry/instrumentation": "0.34.0",
+    "@opentelemetry/instrumentation-express": "0.32.0",
+    "@opentelemetry/instrumentation-graphql": "0.33.0",
+    "@opentelemetry/instrumentation-http": "0.34.0",
+    "@opentelemetry/resources": "1.8.0",
+    "@opentelemetry/sdk-trace-base": "1.8.0",
+    "@opentelemetry/sdk-trace-node": "1.8.0",
```

Comment:
Could this be devDependencies ?

Reply:
A lot of these were not needed as direct dependencies lol. I moved the ones actually needed to dev dependencies.